src/renderer/features/assets/AssetsSettings/model/assets-settings-modal.ts

```diff
@@ -32,6 +33,12 @@ sample({
   target: [getHideZeroBalancesFx, getAssetsViewFx],
 });
 
+sample({
```

Comment:
Seems strange that settings feature cares about starting `price rates request` 🤔
Can we move this into the `Assets page` model?